host-interaction/thread/create/create-thread.yml

```diff
@@ -19,6 +19,12 @@ rule:
         - api: PsCreateSystemThread
         - api: SHCreateThread
         - api: SHCreateThreadWithHandle
+        - api: kernel32.CreateRemoteThread
+        - api: ntdll.RtlCreateUserThread
+        - api: ntdll.NtCreateThread
+        - api: ntdll.NtCreateThreadEx
+        - api: ntdll.ZwCreateThread
+        - api: ntdll.ZwCreateThreadEx
         - and:
           - api: kernel32.GetCurrentProcess
           - api: kernel32.CreateRemoteThread
```

Comment:
We capture all thread creations here now, so can remove this block?

Reply:
I just left it since I thought it has a deeper sense for a certain rule. It is removed in aa94a33.